### PR TITLE
ACS-9897 Handle "tr_status" for blank document

### PR DESCRIPTION
**Pull request by gerardolenski**
- added common DataAttributesUtils to check non-blank content existence

### CODE REVIEW

File: src/test/java/org/alfresco/repo/event/util/DataAttributesUtilsTest.java [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import org.alfresco.repo.event.v1.model.ContentInfo;
+import org.alfresco.repo.event.v1.model.DataAttributes;
+import org.alfresco.repo.event.v1.model.NodeResource;
+import org.alfresco.repo.event.v1.model.Resource;
+
+@ExtendWith(MockitoExtension.class)
+class DataAttributesUtilsTest
```
Comment (cezary-witkowski) [inline src/test/java/org/alfresco/repo/event/util/DataAttributesUtilsTest.java:24]: Consider adding a test case when `ContentInfo::getSizeInBytes` returns null.

**Review by arindam-roy-hld — APPROVED** [review]

File: src/main/java/org/alfresco/repo/event/util/DataAttributesUtils.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+public class DataAttributesUtils
+{
+    private DataAttributesUtils()
+    {}
+
+    /**
+     * Checks within {@link DataAttributes} if it contains not blank content. Content is considered to be present if the resource is a {@link NodeResource} and its {@link ContentInfo#getSizeInBytes()} is greater than 0.
+     */
+    public static boolean containsNotBlankContent(DataAttributes<? extends Resource> dataAttributes)
```
[the remark this answers is not in the document]
Comment (gerardolenski) [reply src/main/java/org/alfresco/repo/event/util/DataAttributesUtils.java:18]: Well. My intention was to add method that will be correctly named in ubiquitous "business" language. I prefer to keep it as it is.
